### PR TITLE
feat(OKX and Phantom): New Wallets

### PR DESCRIPTION
**Pull request by NateDiamond**
Testing: Used tangz-mobile to validate that getWalletAddress worked. Also inserted code to test sendBtc for OKX

### CODE REVIEW

File: src/wallets/wallets.js [hunk 1]
```diff
@@ -60,10 +76,13 @@ export async function sendBtc(walletProvider, address, btcAmount, originator) {
   switch (walletProvider) {
     case HIRO_WALLET:
       return await sendBitcoinFromHiro(btcAmount, address);
+    case OKX_WALLET:
+      return await sendBitcoinFromOkx(btcAmount, address)
     case UNISAT_WALLET:
       return await sendBitcoinFromUnisat(btcAmount, address);
     case XVERSE_WALLET:
       return await sendBitcoinFromXverse(btcAmount, address, originator);
+    case PHANTOM_WALLET:
```
Comment (thaddeusdiamond) [inline src/wallets/wallets.js:85]: Phantom can't send BTC?

File: src/wallets/wallets.js [hunk 2]
```diff
@@ -48,6 +60,10 @@ export async function signPsbt(walletProvider, psbtHex) {
   switch (walletProvider) {
     case HIRO_WALLET:
       return await window.btc?.request('signPsbt', { hex: psbtHex });
+    case OKX_WALLET:
+      return await window.okxwallet?.bitcoin.signPsbt(psbtHex);
+    case PHANTOM_WALLET:
+      return await window?.phantom?.bitcoin.signPSBT(psbtHex);
```
Comment (thaddeusdiamond) [inline src/wallets/wallets.js]: Why the extra ? after window?

**Review by thaddeusdiamond — APPROVED** [review]
LGTM on code, one question on functionality though...

File: src/wallets/wallets.js (hunk 1 above)
Comment (NateDiamond) [reply src/wallets/wallets.js:85]: It can but by PSBT

File: src/wallets/wallets.js (hunk 2 above)
Comment (NateDiamond) [reply src/wallets/wallets.js]: Not needed, fixed.

**Comment by NateDiamond** [comment]
What's the process for releasing these changes? Do I have to run `npm pack` locally and publish to npmjs.com?

**Pull request merged**